pinot-core/src/main/java/org/apache/pinot/core/operator/dociditerators/RangelessBitmapDocIdIterator.java

```diff
@@ -23,7 +23,7 @@
 import org.roaringbitmap.IntIterator;
 
 
-public class RangelessBitmapDocIdIterator implements BlockDocIdIterator {
+public class RangelessBitmapDocIdIterator implements IndexBasedDocIdIterator {
```

Comment:
thats just a marker interface. It better to mark them as indexBased since it has already computed the bitmap otherwise it will be treated as a non-index based iterator.